lib/src/main/kotlin/com/swmansion/starknet/provider/Provider.kt

```diff
@@ -43,14 +43,25 @@ interface Provider {
      */
     fun callContract(call: Call, blockNumber: Int): Request<List<Felt>>
 
+    /**
+     * Calls a contract deployed on StarkNet.
```

Comment:
Perhaps these overloaded method should have a info in their docs that they use default block tag `LATEST`.